project_security_group/models/res_config.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models
+from openerp.osv import fields
+from openerp.tools.safe_eval import safe_eval
+
+
+class ProjectTeamRoleSettings(models.TransientModel):
+    _name = 'project.config.settings'
+    _inherit = ['project.config.settings']
+
+    _columns = {
+        'limit_project_user': fields.boolean(
+            string='Limit project users',
+            help="Limit project user to see only the task and issue he/she is "
+                 "assigned and not all the task of a project. If project user "
+                 "is a team member then has permission to see all tasks and "
+                 "issues of a project."),
```

Comment:
@sergio-incaser @dreispt In v9 team member disappear to the profit of followers.
I am concerned to have new modules based on team members setup knowing that it is already obsolete